.github/workflows/update-caret-version.yml

```
@@ -0,0 +1,37 @@
+name: caret-version
+
+on:
+  push:
+    tags:
+      - v\d\.\d\.\d
+
+jobs:
+  caret-version:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Get latest tag
+        id: get_tag
+        run: |
+          latest_tag=$(curl -s https://api.github.com/repos/tier4/ros2caret/tags | jq -r .[0].name)
```

Why don't you use environmental variable so that this script becomes more general.
```suggestion
          latest_tag=$(curl -s https://api.github.com/repos/${GITHUB_REPOSITORY}/tags | jq -r .[0].name)
```

---

Or, you can just use `${GITHUB_REF_NAME}` variable instead of `latest_tag`